preprocessor/.gitignore

```diff
@@ -0,0 +1 @@
+test_data
```

Comment:
Please move the script to `scripts/preprocessor` so that we have all the scripts in the same place. Note that the validation script should also be moved there since it's currently in the wrong place and @michaelperel has an action item for that from a past pull request review.